doc: Add manual page generation for bitcoin-util

laanwj opened the pull request:
- Add `-version` option to `bitcoin-util`
- Add `bitcoin-util` call to `gen-manpages.sh`
- Add stub manual page `bitcoin-util.1`
- Add install of `bitcoin-util.1` to build system

laanwj committed:
doc: Add manual page generation for bitcoin-util

- Add `-version` option to `bitcoin-util`
- Add `bitcoin-util` call to `gen-manpages.sh`
- Add stub manual page `bitcoin-util.1`
- Add install of `bitcoin-util.1` to build system

adamjonas commented on doc/man/bitcoin-util.1:
```diff
@@ -0,0 +1,5 @@
+.SH NAME
+bitcoin-util \- manual page for bitcoin-util
+
+This is a placefolder file. Please follow the instructions in \fIcontrib/devtools/README.md\fR to generate the manual pages after a release.
```
```suggestion
This is a placeholder file. Please follow the instructions in \fIcontrib/devtools/README.md\fR to generate the manual pages after a release.
```

laanwj replied:
Thanks. We'll want to change this in the other files as well.

laanwj committed:
scripted-diff: Fix typo in stub manual pages

-BEGIN VERIFY SCRIPT-
sed -i 's/placefolder/placeholder/' $(git ls-files doc/man/\*.1)
-END VERIFY SCRIPT-

laanwj commented:
Added scripted-diff commit to fix the typo in the stub manual pages.

fanquake reviewed (approved):
ACK bc99ae77e431740c0eda8a93152ce15600b3852a

```bash
src/bitcoin-util -version            
Bitcoin Core bitcoin-util utility version v21.99.0-bc99ae77e431
```
